### PR TITLE
fix: show domain setup to admins

### PR DESCRIPTION
**Pull request by karthikb351**
Prompts admin users to setup domains on the identity creation

### CODE REVIEW

**Comment by andrewgodwin** [comment]
This looks reasonable - there's a few wording changes I'd want to do but I could easy get those in after a merge.

**Review by andrewgodwin — APPROVED** [review]

File: users/views/identity.py
Comment (karthikb351) [inline users/views/identity.py:377]: This feels a bit ugly to read from the form field choices list. I'm not familiar enough with django formviews to know where to fetch domains so it's passed to both the form class init as well as have it in context.

Comment (andrewgodwin) [reply users/views/identity.py:377]: Yeah, it's not ideal, but not terrible - I'll merge it and then see if I can replace it with a more direct query.

**Pull request merged**
